src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -102,6 +103,21 @@
            (create-field name (display-value value)))
          log-map)])))
 
+(defn- failure-tree [failures]
+  [comps/Tree {:start-collapsed? false
+               :label "Failures"
+               :highlight-ends false
+               :data (prewalk
+                       (fn [elem]
+                         (cond
+                           ;; reorder map elements to message then causedBy
```

Comment:
Yeah I don't prefer this way, but we'd have to modify the `Tree` component to allow specifying some kind of ordering.

The map length here will always be 1 or 2, so my understanding is that my use case is safe ... for now.